packages/codebytes/package.json

```diff
@@ -25,8 +25,11 @@
     "@codecademy/gamut-styles": "*",
     "@codecademy/variance": "*",
     "@emotion/react": "^11.4.0",
-    "@emotion/styled": "^11.3.0"
-  },
+    "@emotion/styled": "^11.3.0",
+    "@loadable/component": "^5.13.1",
+    "react-monaco-editor": "0.34.0",
+    "react-resize-observer": "1.1.1",
+    "monaco-editor": "0.20.0"  },
```

Comment:
The monaco editor should be in its own package, like `@codecademy/monaco-editor`. We want to use it for more than just Codebytes.

Reply:
@JoshuaKGoldberg, we will only be using the simple monaco editor that is local to codebytes as we do static sites repo to avoid scope creep. This will help us get to the finish line faster. If we use the monaco editor that is to be migrated to client-modules in the future, we can handle that as a tech investment item, since it seems like this project required it's own RFC. I'm trying not to pack too much into this migration, and keep them separate! Though we can handle later in another ticket if a team has bandwidth! https://www.notion.so/codecademy/Monaco-editor-as-a-shared-package-1f4484db165b4abc8394c3556451ef6a